go.mod

```diff
@@ -1,14 +1,20 @@
 module github.com/gravitational/trace
 
-go 1.15
+go 1.20
 
 require (
 	github.com/stretchr/testify v1.8.3
-	golang.org/x/net v0.17.0
+	golang.org/x/net v0.33.0
 	google.golang.org/grpc v1.56.3
 )
 
 require (
-	golang.org/x/sys v0.15.0 // indirect
```

Comment:
Not sure why dependabot took out all indirects, but once I upped the Go version it all came back.